integration-tests/docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM quay.io/redhatqe/cypress:9.6.0
+
+USER root:root
```

Comment:
I'm not a docker expert but using a root user is not a good idea ([see this](https://security.stackexchange.com/questions/106860/can-a-root-user-inside-a-docker-lxc-break-the-security-of-the-whole-system)).
Maybe we can use a non-root user ([see this](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#user)) by adding following commands in place of line 3:
`RUN groupadd -r hacdev && useradd -r -g hacdev hacdev`
`USER hacdev:hacdev`

Reply:
I'm neither a docker expert :wink: But from what I see, we need root permissions to install a new software in this container. When I put  your commands instead the `USER root:root` it fails on:
```
groupadd: Permission denied.
```
If I do that like a root and then switch to a new user, I'm getting an error during installation of `which` :
```
error: Failed to create: /var/cache/yum/metadata
```
So I guess that way the `hacdev` user doesn't have enough permissions. 
Anyway, this root user is used for **building** the container, so I would say it may be OK to leave the root here.

Reply:
Sorry for not testing locally before adding my previous comment.
One way is to install all the relevant packages as a root user and then switch to a non-user after that ([I referred this](https://stackoverflow.com/a/54269736)). I tested locally by adding 
> RUN groupadd -r hacdev && useradd -r -g hacdev hacdev
USER hacdev:hacdev

to the end of the Dockerfile, the image gets built successfully with the root staying in its original place. I'm also curious about what @jrichter1 thinks about this.

Reply:
AFAIK the test commands are called with `-u 0`, which probably is root anyway. Would be nice to have a non root user, maybe it would eliminate the need to `chmod a+rwx` for the build artifacts.

Reply:
let root do the installation, you need the permissions for that, but use a non root to run the tests